src/pynxtools/dataconverter/readers/multi/reader.py

```diff
@@ -137,12 +139,42 @@ def try_convert(value: str) -> Union[str, float, int, bool]:
 
         return value
 
-    if not isinstance(value, str) or ":" not in value:
+    def parse_config_value(value: str) -> Tuple[str, Any]:
+        # Regex pattern to match @prefix:some_string
+        pattern = r"!?(@\w+)(?::(.*))?"
+        match = re.match(pattern, value)
+        if match:
+            prefix, suffix = match.groups()
+            if suffix is None:
+                suffix = ""
+            return (prefix, suffix)
+        else:
+            return ("", value)
```

Comment:
Can be a bit simpler
```suggestion
        prefixes = re.findall(pattern, value)
        if not prefixes:
            return ("", value)
        return prefixes[0]
```

Reply:
done